.github/workflows/selftest-linux.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     name: Local linux self-test
     strategy:
       matrix:
-        distro: [ ubuntu, suse, centos ]
+        distro: [ ubuntu, suse, rockylinux ]
```

Comment:
Keep in mind that doing this will break the interface for the action (not like we use the non-defaults anywhere, but still).
Moreover it will remove the option to specify `centos` anywhere, even for supported versions.

Are we sure we want to do this, as opposed to just shim the `centos:8` image to `rockylinux:8` in the `qualify_distro` and `helper_name` images?

Reply:
I would prefer to have pipelines failing because we have hardcoded distros instead of using the defaults (which are in fact all the distros we support)

Reply:
Another example, if tomorrow we want to deprecate Debian 8 we can simply remove it from here instead of all the pipelines.

Reply:
Yes this is breaking as you mentioned , I didn't find any integration using the `distro` so shouldn't affect any integration.
On the other side is true that I'm removing `centos7` with this too, but i would assume that it could brake any time in the near future.